LICENSE

```diff
@@ -1,22 +1,21 @@
-Copyright (c) 2015-2019 Matteo Collina, http://matteocollina.com
```

Comment:
That's not how this works. Please retain this line and place it after the new one.

Reply:
Thanks for pointing out. :)